core/src/main/java/io/questdb/std/str/DirectByteCharSink.java

```diff
@@ -116,4 +115,15 @@ public int size() {
     public String toString() {
         return Chars.stringFromUtf8Bytes(this);
     }
+
+    @Override
+    public DirectByteCharSink put(CharSequence cs) {
+        // TODO: Improve UTF-8 compatibility.
```

Comment:
i don't think this `todo` will ever be actionable and should be removed. `byte` in the name is a clue to not use this for non-ascii java UTF16 stuff

Reply:
we do not merge code with `todos`, i don't think it is appropriate to cut releases with `todo` in them. Future work should be tracked outside of `todo`. @puzpuzpuz should have separate todo list other that "todos" in `master`

Reply:
I'll coordinate with Andrei on this and make sure what needs doing is tracked and remove the comment.

Reply:
Done.